content/en/docs/kubernetes/getting-started.md

```diff
@@ -0,0 +1,276 @@
+using a Kubernetes operator, see [OpenTelemetry Operator](../operator/), but
+this guide will focus on the Helm chart.
+
+## Preparation
+
+This guide will assume the use of a [Kind cluster](https://kind.sigs.k8s.io/),
+but you're free to use any Kubernetes cluster that you deem fit.
+
+Assuming you already have
+[Kind installed](https://kind.sigs.k8s.io/#installation-and-usage), create a new
+kind cluster:
+
+```sh
+kind create cluster
+```
+
+Assuming you already have [Helm installed](https://helm.sh/docs/intro/install/),
+add the OpenTelemetry Collector Helm chart so it can be installed later.
+
+```sh
+helm repo add open-telemetry https://open-telemetry.github.io/opentelemetry-helm-charts
+```
+
+## Daemonset Collector
+
+The first step to collecting Kubernetes telemetry is to deploy a daemonset
+instance of the collector to gather telemetry related to nodes and workloads
+running on those node. A daemonset is used to guarantee that this instance of
+the collector is installed on all nodes. Each instance of the collector in the
+daemonset will collect data only from the node on which it is running.
+
+This instance of the collector will use the following components:
+
+- [OTLP Receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver):
+  collects application traces, metrics and logs.
+- [Kubernetes Attributes Processor](../collector/components#kubernetes-attributes-processor):
+  adds Kubernetes metadata to incoming application telemetry.
+- [Kubeletstats Receiver](../collector/components#kubeletstats-receiver): pulls
+  node, pod, and container metrics from the API server on a kubelet.
+- [Filelog Receiver](../collector/components#filelog-receiver): collects
+  Kubernetes logs and application logs written to stdout/stderr.
+
+Lets break these down.
+
+**OTLP Receiver**
+
+The
+[OTLP Receiver](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver)
+is the best solution for collecting traces, metrics, and logs in the
+[OTLP format](https://opentelemetry.io/docs/specs/otel/protocol/). If you are
+emitting application telemetry in another format there is a good chance that
+[the Collector has a receiver for it](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver),
+but for this tutorial we'll assume the telemetry is formatted in OTLP.
+
+Although not a requirement, it is a common practice for applications running on
+a node to emit their traces, metrics, and logs to a collector running on the
+same node, normally by using the . This keeps network interactions simple and
+allows easy correlation of Kubernetes metadata using the k8sattributes
+processor.
+
+**Kubernetes Attributes Processor**
+
+The
+[Kubernetes Attributes Processor](../collector/components#kubernetes-attributes-processor)
+is a must have component in any collector receive telemetry from kubernetes
```

Comment:
```suggestion
is a must have component in any collector to receive telemetry from kubernetes
```

Reply:
I'd prefer not to use `required` here since it technically isn't require, only strongly suggested.

Reply:
"Must have" can be interpreted as a synonym for "required". Please select a different word choice then like "recommended".

Reply:
updated